<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cat文本编辑器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>标准</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（●）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（○）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（□）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（1..2..3..）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（a..b..c..）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（A..B..C..）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（i..ii..iii..）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号（I..II..III...）</string>
     </property>
    </item>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1161</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveOther"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(D)</string>
     </property>
     <addaction name="inclineAction"/>
     <addaction name="underlineAction"/>
     <addaction name="blodAction"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="leftAction"/>
     <addaction name="centerAction"/>
     <addaction name="rightAction"/>
     <addaction name="justifyAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu_2"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗体(&amp;W)</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="titleAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="printAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="colorAction"/>
   <addaction name="inclineAction"/>
   <addaction name="underlineAction"/>
   <addaction name="blodAction"/>
   <addaction name="separator"/>
   <addaction name="leftAction"/>
   <addaction name="centerAction"/>
   <addaction name="rightAction"/>
   <addaction name="justifyAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/newfile.png</normaloff>:/images/newfile.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="statusTip">
    <string>新建一个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/openfile.png</normaloff>:/images/openfile.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="statusTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/savefile.png</normaloff>:/images/savefile.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="statusTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveOther">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/saveother.png</normaloff>:/images/saveother.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)...</string>
   </property>
   <property name="statusTip">
    <string>另存为...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="printAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(P)...</string>
   </property>
   <property name="statusTip">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/printpreview.png</normaloff>:/images/printpreview.png</iconset>
   </property>
   <property name="text">
    <string>打印预览...</string>
   </property>
   <property name="statusTip">
    <string>打印预览</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/revoke.png</normaloff>:/images/revoke.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="statusTip">
    <string>撤销一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(R)</string>
   </property>
   <property name="statusTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/shear.png</normaloff>:/images/shear.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="statusTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="statusTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="statusTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色(C)...</string>
   </property>
   <property name="statusTip">
    <string>颜色面板</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="inclineAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/incline.png</normaloff>:/images/incline.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(I)
</string>
   </property>
   <property name="statusTip">
    <string>字体倾斜</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="underlineAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)
</string>
   </property>
   <property name="statusTip">
    <string>字体下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="leftAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/leftjustifying.png</normaloff>:/images/leftjustifying.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(L)
</string>
   </property>
   <property name="statusTip">
    <string>左对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Center.png</normaloff>:/images/Center.png</iconset>
   </property>
   <property name="text">
    <string>居中(C)
</string>
   </property>
   <property name="statusTip">
    <string>居中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="rightAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rightjustifying.png</normaloff>:/images/rightjustifying.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(R)
</string>
   </property>
   <property name="statusTip">
    <string>右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="justifyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/both.png</normaloff>:/images/both.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐(J)
</string>
   </property>
   <property name="statusTip">
    <string>两端对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(O)</string>
   </property>
   <property name="statusTip">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
   <property name="statusTip">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="titleAction">
   <property name="text">
    <string>平铺(T)</string>
   </property>
   <property name="statusTip">
    <string>平铺</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>层叠(C)</string>
   </property>
   <property name="statusTip">
    <string>层叠</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个(X)</string>
   </property>
   <property name="statusTip">
    <string>下一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>前一个(V)</string>
   </property>
   <property name="statusTip">
    <string>前一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="statusTip">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="blodAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Bold.png</normaloff>:/images/Bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
   <property name="statusTip">
    <string>加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
